benchmarks/work_precision_sets/time_vs_abstols.py

```diff
@@ -0,0 +1,126 @@
+import pybamm
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+parameters = [
+    "Marquis2019",
+    "Prada2013",
+    "Ramadass2004",
+    "Chen2020",
+]
+models = ["SPM", "DFN"]
```

Comment:
You can actually create a list of PyBaMM models. You can also pass their name (to abbreviate it). This way you can get rid of lines 34-37.

```python3
models = [pybamm.lithium_ion.SPM(name="SPM"), pybamm.lithium_ion.DFN(name="DFN")]
```

You can then access the name as `models[i].name`.